less/transferwise.less

```diff
@@ -142,3 +141,27 @@ li .circle {
     color: @brand-smoke-minus-20;
   }
 }
+
+/**
+ * Notification center
+ */
+
+@media screen and (max-width: 767px) {
+  .notification-center {
```

Comment:
Notification center isn't a generic component, it's a domain specific implementation.  We should add a modifier class to the dropdown.less.  Something like 'dropdown-overlay' or 'dropdown-native' such that it can be used on any dropdown that wants this behaviour.

Reply:
Actually, could we also put the breakpoint in the class.  So if this applies on 'sm' and below, it would be something like
```
@media screen and (max-width: @screen-sm-max) {
  .dropdown-sm-overlay .open {
    ...
  }
}
```
It's more future proof if we wanted to do this at different breakpoints.  I'm not sure 767px is screen-sm-max, so I would check.